back/spec/services/local_project_copy_service_spec.rb

```diff
@@ -312,4 +312,50 @@
       end
     end
   end
+
+  if CitizenLab.ee?
+    describe 'when source project has associated content builder layout' do
+      let(:layout) { create(:layout, code: 'project_description') }
+
+      it 'copies content builder layout' do
+        copied_project = service.copy(layout.content_buildable)
+
+        expect(copied_project.content_builder_layouts.first
+          .as_json(only: %i[content_buildable_type code enabled]))
+          .to eq(layout.as_json(only: %i[content_buildable_type code enabled]))
+      end
+
+      it 'copies associated layout images as new images, associated with copied layout' do
+        image1 = create(:layout_image)
+        image2 = create(:layout_image)
+
+        image_codes = { code1: image1.code, code2: image2.code }
+        craftjs_str = ERB.new(File.read('spec/fixtures/craftjs_layout_with_2_images.json.erb'))
+          .result(image_codes.instance_eval { binding })
```

Comment:
Looks good! You can also use `result_with_hash` instead of `result`, it looks a little simpler https://stackoverflow.com/a/44266619

Reply:
Nice! That helped me clean up that test even more.